writer.go

```diff
@@ -443,7 +443,7 @@ func (p *MediaPlaylist) Encode() *bytes.Buffer {
 
 	head := p.head
 	count := p.count
-	for i := uint(0); i < p.winsize && count > 0; count-- {
+	for i := uint(0); (i < p.winsize || p.winsize == 0) && count > 0; count-- {
```

Comment:
`p.Closed || i < p.winsize && count > 0` means `(p.Closed || i < p.winsize) && count > 0` ?

lgtm, but I feel like having the demand that wants to show everything in live.